pkg/plugins/builtin/http/http.go

```diff
@@ -68,6 +58,7 @@ type parameter struct {
 type auth struct {
 	Basic  authBasic `json:"basic"`
 	Bearer string    `json:"bearer"`
+	MTLS   mTLS      `json:"mTLS"`
```

Comment:
Hah, too bad Go struct fields naming rules got in your way 😄

Reply:
Sad developer is sad.